styles.css

```diff
@@ -454,415 +560,478 @@ body {
     display: none;
   }
 
-  .drop-down-menu {
-    z-index: 10;
-    color: #fff;
-    background-color: #6070ff;
-    height: 110vh;
-    width: 100%;
-    mix-blend-mode: multiply;
-    backdrop-filter: blur(7px);
-    position: fixed;
+  .desktop-popup {
+    display: none;
   }
 
-  .hidden-menu {
+  .desktop-work {
     display: none;
   }
 
-  .hamburger-icon {
-    margin-right: 5%;
-    margin-top: 3%;
+  .mobile-popup {
+    height: 90%;
+    width: 80%;
+    background: #fff;
+    z-index: 2;
+    position: fixed;
+    top: 0;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5%;
+    margin: 5%;
+    margin-top: 10%;
   }
 
-  .show {
-    display: block;
+  .popup-line {
+    width: 90%;
+    height: 2px;
+    background-color: #ebecf0;
+    margin: 5%;
   }
 
-  .fa-bars {
-    font-size: 20px;
-    color: #6070ff;
+  .popup-project-topic {
+    margin: 0;
   }
 
-  .nav-dropdown {
-    list-style-type: none;
-    margin-top: 50px;
+  .popup-close {
+    position: absolute;
+    top: 3%;
+    right: 5%;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    color: #67798e;
+    cursor: pointer;
   }
 
-  .nav-dropdown-item-link {
-    text-decoration: none;
-    color: #fff;
-    font-weight: 600;
-    font-size: 32px;
-    line-height: 62px;
-    gap: 30px;
-    font-family: 'Poppins', sans-serif;
+  .popup-buttondiv {
+    display: flex;
+    padding: 0%;
   }
 
-  .dropdown-x-icon {
-    float: right;
-    margin-right: 6.5%;
-    margin-top: 6.5%;
+  .popupbutton-child {
+    color: #396df2;
+    font-weight: 500;
+    font-size: 15px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #396df2;
+    margin-top: 0%;
+    padding: 2.7%;
   }
```

Comment:
- [ ] [OPTIONAL] :+1: You have done an amazing job implementing feedback from previous reviewers, and I think to make your project even more aesthetic and improve user experience for mobile users, it will be worth to make some small changes to improve the present UI. The buttons seem to big for the available space and it's into one another in a not so good way. Here's a screenshot to help you understand what I meant :point_down: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/218120689-f485f3d9-a80a-4ba5-b91a-cb9dbc15a10c.png)|![image](https://user-images.githubusercontent.com/99715640/218120790-9dc09e42-07f1-44f2-8fa3-73605c35c972.png)|